addons/l10n_latam_invoice_document/wizards/account_move_reversal_view.xml

```diff
@@ -9,10 +9,11 @@
             <form>
                 <field name="l10n_latam_use_documents" invisible="1"/>
                 <field name="l10n_latam_sequence_id" invisible="1"/>
+                <field name="move_id" invisible="1"/>
             </form>
             <field name="date" position="before">
                 <field name="l10n_latam_document_type_id" attrs="{'invisible': ['|', ('l10n_latam_use_documents', '=', False), ('refund_method', '=', 'refund')], 'required': [('l10n_latam_use_documents', '=', True), ('refund_method', '!=', 'refund')]}" options="{'no_open': True, 'no_create': True}"/>
-                <field name="l10n_latam_document_number" attrs="{'invisible': ['|', ('l10n_latam_sequence_id', '!=', False), ('refund_method', '=', 'refund')], 'required': [('l10n_latam_sequence_id', '=', True), ('refund_method', '!=', 'refund')]}"/>
+                <field name="l10n_latam_document_number" attrs="{'invisible': ['|', ('l10n_latam_sequence_id', '!=', False), ('refund_method', '=', 'refund')], 'required': [('l10n_latam_sequence_id', '=', False), ('l10n_latam_use_documents', '=', True), ('refund_method', '!=', 'refund')]}"/>
```

Comment:
For the invisible I cannot agree, because it will be visible for other countries where there is no latam_use_documents.  (even if it was like that before)